components/04-indicators/font-icon/font-icon.hbs

```diff
@@ -0,0 +1 @@
+<i class="{{ icon }}" aria-hidden="true"></i>
```

Comment:
Ik vermoed dat DSO voorkeur heeft dat we het `span` element gebruiken. In hoeverre is die component anders dan https://www.dso-toolkit.nl/7.0.1/components/detail/icon.html ?

Reply:
Waarom `span`?

Reply:
Klopt, maar toen wij de toolkit overgedragen kregen van DSO werden alle iconen met `span` gemaakt. Ik heb er nooit naar gevraagd waarom, maar ik ben ooit eens dit bij FA tegengekomen:

>Font Awesome is designed to be used with inline elements and we recommend sticking with a consistent HTML element to reference them by in your project. We like the <i> tag for brevity and for the fact that most folks are using <em></em> for emphasized/italicized semantic text these days. If that’s not your cup of tea, using a <span> is more semantically correct.
/ https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use

Ik ben er destijds vanuit gegaan dat er is over nagedacht dat er een voorkeur was voor `span`, omdat dit semantisch correct(er ?) is. Maar het is mij om het even, als we maar door het hele project de `i` gaan gebruiken en dat er niet twee smaken worden gedocumenteerd/gepresenteerd